book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+# Collaborative Code Review
+
```

Comment:
I think this subchapter would benefit from an introductory paragraph - what is "Collaborative Code Review" and when would researchers or readers of this book encounter it? What will the rest of the chapter tell them about collaborative code review?